erica_app/erica/request_processing/eric_mapper.py

```diff
@@ -0,0 +1,197 @@
+            values.get('person_a_has_merkzeichen_h'),
+            values.get('person_a_has_merkzeichen_ag')
+        ]
+        if values.get('person_a_requests_fahrkostenpauschale'):
+            if any(merkzeichen_for_higher_fahrkostenpauschale):
+                values['person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad'] = True
+            elif values.get('person_a_disability_degree') >= 80 or (values.get('person_a_has_merkzeichen_g') and values.get('person_a_disability_degree') >= 70):
+                values['person_a_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80'] = True
+        return values
+
+    @root_validator(pre=True)
+    def set_person_b_merkzeichen_bl_tbl_h_pflegegrad(cls,values):
+        merkzeichen_values = [
+           values.get('person_b_has_pflegegrad'),
+           values.get('person_b_has_merkzeichen_bl'),
+           values.get('person_b_has_merkzeichen_tbl'),
+           values.get('person_b_has_merkzeichen_h')
+        ]
+        if any(merkzeichen_values):
+            values['person_b_pauschbetrag_has_merkzeichen_bl_tbl_h_pflegegrad'] = True
+        return values
+
+    @root_validator(pre=True)
+    def set_person_b_merkzeichen_g_ag(cls, values):
+        merkzeichen_values = [
+           values.get('person_b_has_merkzeichen_g'),
+           values.get('person_b_has_merkzeichen_ag')
+        ]
+        if any(merkzeichen_values):
+            values['person_b_pauschbetrag_has_merkzeichen_g_ag'] = True
+        return values
+
+    @root_validator(pre=True)
+    def set_person_b_fahrkostenpauschale(cls, values):
+        merkzeichen_for_higher_fahrkostenpauschale = [
+            values.get('person_b_has_pflegegrad'),
+            values.get('person_b_has_merkzeichen_bl'),
+            values.get('person_b_has_merkzeichen_tbl'),
+            values.get('person_b_has_merkzeichen_tbl'),
+            values.get('person_b_has_merkzeichen_h'),
+            values.get('person_b_has_merkzeichen_ag')
+        ]
+        if values.get('person_b_requests_fahrkostenpauschale'):
+            if any(merkzeichen_for_higher_fahrkostenpauschale):
+                values['person_b_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad'] = True
+            elif values.get('person_b_disability_degree') >= 80 or (values.get('person_b_has_merkzeichen_g') and values.get('person_b_disability_degree') >= 70):
+                values['person_b_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80'] = True
+        return values
+
+    @validator('person_a_dob', 'person_b_dob', 'familienstand_date', pre=True)
+    def convert_datetime_to_d_m_y(cls, v):
+        if v:
+            return v.strftime('%d.%m.%Y')
+        return None
+
+    @validator('person_a_pauschbetrag_disability_degree', 'person_b_pauschbetrag_disability_degree')
+    def set_disability_degree_only_if_above_20(cls, v):
+        if v and v < 20:
+            return None
+        return v
```

Comment:
Uh uh uh: mini optimization:

`return v.strftime('%d.%m.%Y') if v else None` and
`return None if v and v < 20 else v`